pkg/sync/v9/client.go

```diff
@@ -294,6 +294,12 @@ func (s *sync) isReconcileProtected(existing, o *unstructured.Unstructured) bool
 			(o.GroupVersionKind().GroupKind() == schema.GroupKind{Group: "rbac.authorization.k8s.io", Kind: "ClusterRoleBinding"}) {
 			return true
 		}
+		// log analytics agent data collection configuration
```

Comment:
I think it would make more sense to unset the "owned-by-sync-pod" annotation in the CM ?

Reply:
sounds good

Reply:
@thekad discussed this with the team and we decided to go with the `openshift.io/reconcile-protect` annotation since the `azure.openshift.io/owned-by-sync-pod` label should not be touched by the customer.